DEVELOPER.md

```diff
@@ -23,13 +23,14 @@ Current availability versions are:
 
 ```
 furyctl
-├── install : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
+├── init : Downloads Furyfile.yml and kustomization.yaml from the distribution repository to the current directory.
+├── vendor : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
 ├── printDefault : prints a Furyfile example
 ├── help
 └── version
 ```
 
-Write a [`Furyfile.yml`](Furyfile.yml) in the root of your project and then simply run `furyctl install`.
+Write a [`Furyfile.yml`](Furyfile.yml) in the root of your project and then simply run `furyctl vendor`.
 It will download what you have specified in the `vendor` directory in your project root.
```

Comment:
I'd change this line, to something like this:
`It will download modules specified inside 'Furyfile.yaml' to the 'vendor' directory.`